src/components/vm/nics/vmNicsCard.jsx

```diff
@@ -124,9 +145,7 @@ export class VmNetworkTab extends React.Component {
 
         // Load the interface addresses list when the tab mounts
         vmInterfaceAddresses(this.props.vm.connectionName, this.props.vm.id)
-                .then(ifaces => {
-                    this.setState({ interfaceAddress: ifaces[0] });
-                }, ex => {
+                .then(res => this.setState({ domifaddress: res[0] }), ex => {
                     this.props.onAddErrorNotification({
```

Comment:
What does it mean if it cannot find any IP address? Would that in extent also mean that VM has no network connectivity? If not, then maybe showing the error message like this is too aggressive
![Screenshot from 2021-05-04 12-58-26](https://user-images.githubusercontent.com/42733240/116994011-7d4c5500-acd8-11eb-9dea-ba538730855e.png)
Since it could make user believe that's some big problem which needs to be fixed. But if it just means that they will not be able to see the IP address in vNIC tab, and the VM will function normally, maybe we could just print the error to `console.error()` 

Again, I suggest this only if this error doesn't cause VM to lose network connectivity. If it does, then it's important and let's keep it like this.

Reply:
To be honest I was not able to create this message in any way but to `fake` the method names to make them fail. How did you make it fail?

Basicaly any NIC type I added, ended up working with at least one of these methods.

Reply:
Of course for the qemu-guest-agent to work we need the agent inside guest, which is present for cloud images but not generally.
But ARP tables seemed very generic.

Reply:
I don't know why that one in my screenshot fails, but I found out you can get all these errors also if you run our test VM `./bots/vm-run fedora-33 --maintain`, and then thru the UI add some network interface to it (i.e. Userspace SLIRP stack) and reload the page (the fact that you have to reload the page to get IP info is also a problem)

Reply:
Good catch, so empty list returned from arp or any of these sources should not be considered an error.
Fixed this, and will repost soon.